### PR TITLE
Ubuntu 17.10 LE ppc image

### PR DESCRIPTION
**Pull request by ramereth**
These isn't a BE build for 17.10 so only building LE.

### CODE REVIEW

**Comment by ramereth** [comment]
[Ubuntu 14.04 / 17.10 ppc](https://trello.com/c/lu4hFm0F/673-ubuntu-1404-1710-ppc)

File: ubuntu-17.10-ppc64le-openstack.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      ],
+      "accelerator": "kvm",
+      "boot_wait": "6s",
+      "disk_size": 4096,
+      "headless": true,
+      "vnc_bind_address": "0.0.0.0",
+      "http_directory": "http",
+      "iso_checksum": "da66eb3aeebaff2073b648c16a79dc9368fd11f36d049847943b6951524c47aa",
+      "iso_checksum_type": "sha256",
+      "iso_url": "{{user `mirror`}}/releases/17.10/release/ubuntu-17.10.1-server-ppc64el.iso",
```
Comment (doublej472) [inline ubuntu-17.10-ppc64le-openstack.json]: I don't think this file exists? When I look at the `sha256sums` file I can see that `ubuntu-17.10-server-ppc64el.iso` is the name file, just with a different name.

**Comment by doublej472** [comment]
@ramereth should we skip over this release and move to 18.04, which is an LTS release?

**Comment by doublej472** [comment]
@ramereth [example of a ticket requesting 18.04 support](https://support.osuosl.org/Ticket/Display.html?id=30092)

**Pull request closed without merging**